tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "lib": ["ESNext"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "target": "ESNext",
+    "target": "ES2018",
```

Comment:
I'm guess from the version of Node listed on `serverless.yml`, we're running Node v12? If so, we should switch the target to something less.

My motivation here was just to get optional chaining working (e.g. you can call `a?.b?.c` instead of `a && a.b && a.b.c`). If you specify `ESNext`, TypeScript will assume you're targeting the latest version of Node and since optional chaining is supported there, it won't transpile any optional chaining in your code. But optional chaining _isn't_ supported in Node v12. By telling TypeScript we're targeting ES2018, it'll convert the optional chaining to something that is supported on olde version of Node.